### PR TITLE
Working

### PR DESCRIPTION
**Pull request by BSGreaves**

### CODE REVIEW

**Comment by BSGreaves** [comment]
@drdreed This one's ready too.
It's a little nasty to "grade" because of all the steps we had to do. Sorry.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<!doctype html>
```
Comment (DevDreed) [inline index.html:1]: <!DOCTYPE html> should be like this.

Comment (DevDreed) [reply index.html:1]: `<!DOCTYPE html>`

**Review by DevDreed — CHANGES_REQUESTED** [review]

**Comment by morecallan** [comment]
@BSGreaves nice catch by drdreed but no need to change this for this exercise. technically the tag is case insensitive, it just shows in caps to emphasize importance

**Pull request merged**